Library/AuthorizationHandler.ts

```diff
@@ -1,32 +1,42 @@
 ﻿import http = require("http");
 import https = require("https");
-import azureCoreAuth = require("@azure/core-auth");
-import azureCore = require("@azure/core-rest-pipeline");
+import { TokenCredential } from "@azure/core-auth";
+import { PipelineRequest, PipelineResponse, PipelinePolicy } from "@azure/core-rest-pipeline";
+import Logging = require("./Logging");
 
 const applicationInsightsResource = "https://monitor.azure.com//.default";
 
+let azureCore: any;
+try {
+  azureCore = require("@azure/core-rest-pipeline");
```

Comment:
This package is needed to send the actual telemetry to Breeze, if not available or not functional the SDK would do nothing, so this is a not necessary check.